project/dexter-peer-review/src/java/com/samsung/sec/dexter/peerreview/plugin/DexterPeerReviewPlugin.java

```diff
@@ -0,0 +1,311 @@
+ * 
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.samsung.sec.dexter.peerreview.plugin;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.log4j.Logger;
+
+import com.google.common.collect.Lists;
+import com.samsung.sec.dexter.core.analyzer.AnalysisConfig;
+import com.samsung.sec.dexter.core.analyzer.AnalysisEntityFactory;
+import com.samsung.sec.dexter.core.analyzer.AnalysisResult;
+import com.samsung.sec.dexter.core.analyzer.IAnalysisEntityFactory;
+import com.samsung.sec.dexter.core.checker.CheckerConfig;
+import com.samsung.sec.dexter.core.config.DexterConfig;
+import com.samsung.sec.dexter.core.config.DexterConfig.LANGUAGE;
+import com.samsung.sec.dexter.core.defect.Defect;
+import com.samsung.sec.dexter.core.defect.Occurence;
+import com.samsung.sec.dexter.core.plugin.IDexterPlugin;
+import com.samsung.sec.dexter.core.plugin.PluginDescription;
+import com.samsung.sec.dexter.core.plugin.PluginVersion;
+
+import net.xeoh.plugins.base.annotations.PluginImplementation;
+
+@PluginImplementation
+public class DexterPeerReviewPlugin implements IDexterPlugin {
+	private final static Logger logger = Logger.getLogger(DexterPeerReviewPlugin.class);
+
+	public final static String PLUGIN_NAME = "dexter-peerreview";
+	public final static PluginVersion PLUGIN_VERSION = new PluginVersion("0.0.1");
+	private static PluginDescription PLUGIN_DESCRIPTION;
+
+	private IAnalysisEntityFactory analysisEntityFactory = new AnalysisEntityFactory();
+
+	final String criticalRegExp = "(?i).*\\[.*cri.*\\] | (?i).*\\[.*critical.*\\]";
+	final String majorRegExp = "(?i).*\\[.*maj.*\\] | (?i).*\\[.*major.*\\]";
+	final String simpleRegExp = ".*(DPR:)";
+
+	public DexterPeerReviewPlugin() {
```

Comment:
Actually, if you don't use constructor with any parameter(s),
you don't need to define constructor.
Just style issue.